aries-site/src/pages/landing/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
This will still be creating a route at aries.hpe.design/landing, but the actual route should just be '/'. If we don't want an unused route, then this code should all be placed in the root index.js file. If not, we should make sure to add the canonical url to the head.

Reply:
@taysea , I agree, I'll fix it on the next PR that will actually address the Landing page implementation.
For the moment this change won't have any impact because we are not directing to the Landing page in any way.